internal/pkg/service/buffer/storage/local/writer/writer.go

```diff
@@ -82,18 +82,26 @@ func (v *Volume) NewWriterFor(slice *storage.Slice) (w SliceWriter, err error) {
 		return nil, err
 	}
 
+	// Get file info
+	stat, err := file.Stat()
+	if err != nil {
+		return nil, err
+	}
+
 	// Allocate disk space
-	if size := slice.LocalStorage.AllocateSpace; size != 0 {
-		if ok, err := v.config.allocator.Allocate(file, size); ok {
-			logger.Debugf(`allocated disk space "%s"`, size)
-		} else if err != nil {
-			// The error is not fatal
-			logger.Errorf(`cannot allocate disk space "%s", allocation skipped: %s`, size, err)
+	if isNew := stat.Size() == 0; isNew {
+		if size := slice.LocalStorage.AllocateSpace; size != 0 && isNew {
```

Comment:
to další `&& isNew` asi už tady nepotřebuješ, když je to v tom parent if, zajímavé že na to go neupozorní v PHP by to neprošlo.

Reply:
dik [fixnute](https://github.com/keboola/keboola-as-code/pull/1425/commits/4c0c020ddf91d982955e6384252ecdbb58b40370)

aktualna konfiguracia [golangci-lint](https://golangci-lint.run/usage/linters/) to nenajde